### PR TITLE
Add switch to hide answer choice pills

### PR DESCRIPTION
**Pull request by nedredmond**
## Summary:

Adds a switch to hide/show the new answer choice pills.

Issue: LC-1420

https://github.com/Khan/perseus/assets/23404711/2b98261b-2c1c-4ae3-95f5-e9b38456d367

### CODE REVIEW

File: packages/perseus/package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
         "@khanacademy/wonder-blocks-popover": "^3.0.0",
         "@khanacademy/wonder-blocks-progress-spinner": "^2.0.17",
         "@khanacademy/wonder-blocks-spacing": "^3.0.5",
+        "@khanacademy/wonder-blocks-switch": "^1.1.6",
```
Comment (jeremywiebe) [inline packages/perseus/package.json:48]: This dependency should also be added to the `peerDependencies` section.

File: packages/perseus/src/widgets/label-image/marker.tsx [hunk 2]
```diff
@@ -54,8 +55,8 @@ export default class Marker extends React.Component<Props> {
         // keyboard focus is given to the marker.
         const isSelected = showSelected || focused;
 
-        let innerIcon;
-        let iconStyles;
+        let innerIcon: React.ReactElement | undefined;
+        let iconStyles: Array<CSSProperties | Falsy>;
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/label-image/marker.tsx:59]: **suggestion::* You can use the `StyleType` from `@khanacademy/wonder-blocks-core` here

**Review by jeremywiebe — COMMENTED** [review]
I'll hold on approving until Nicole has a chance to look.  :)

File: packages/perseus/src/widgets/label-image/marker.tsx (hunk 2 above)
Comment (nedredmond) [reply packages/perseus/src/widgets/label-image/marker.tsx]: thanks!

File: packages/perseus/package.json (hunk 1 above)
Comment (nedredmond) [reply packages/perseus/package.json:48]: thanks!

**Pull request merged**